src/components/recently-viewed.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import { useHover } from 'use-events';
+import { Flex, Box, Text, FlexProps, BoxProps } from '@blockstack/ui';
+
+import { Tag } from '@components/tags';
+import { truncateMiddle } from '@common/utils';
+import { Transaction } from '@models/transaction.interface';
+import Link from 'next/link';
+
+dayjs.extend(relativeTime);
+
+interface RecentlyViewedProps extends FlexProps {
+  transactions: Transaction[];
+}
+
+export const RecentlyViewed = ({ transactions, ...rest }: RecentlyViewedProps) => {
```

Comment:
There is a box-shadow applied to this in figma: `box-shadow: 0px 8px 16px rgba(27, 39, 51, 0.08);`